README.md

```diff
@@ -65,28 +75,15 @@ for part in mtd[0123] ; do
 done
 ```
 
-Then, copy the resulting files using scp to your host.
-
-## Backup the Original/Vendor Bootchain under OpenWrt ##
-
-Connect to the device via SSH and enter the following commands:
-
-```
-mkdir /tmp/boot_backup
-mount -t ubifs /dev/ubi0_3 /tmp/boot_backup
-```
-
-Then, copy the files under /tmp/boot_backup using scp to your host. These files can also be used in emergency case for reflashing via [JTAG](https://openwrt.org/toh/linksys/e8450#jtag)
+Then, copy the resulting files using *scp* to your host.
 
-## Restoring the Vendor/Official Firmware ##
+## Restoring the vendor/official firmware ##
 
 1. Flash `openwrt-mediatek-mt7622-linksys_e8450-ubi-initramfs-recovery.itb` (note that this file doesn't have the word _installer_ in it's filename)
```

Comment:
Since [v0.6.1](https://github.com/dangowrt/linksys-e8450-openwrt-installer/releases/tag/v0.6.1) it's enough to just enter recovery regularly, no need to flash anything at this point.